src/main/java/io/kuzzle/sdk/core/KuzzleDataCollection.java

```diff
@@ -70,8 +70,8 @@ public KuzzleDataCollection(@NonNull final Kuzzle kuzzle, @NonNull final String
    * @param filter   the filter
    * @param listener the listener
    */
-  public void advancedSearch(final JSONObject filter, final KuzzleResponseListener<KuzzleDocumentList> listener) {
-    this.advancedSearch(filter, null, listener);
+  public void search(final JSONObject filter, final KuzzleResponseListener<KuzzleDocumentList> listener) {
```

Comment:
I confirm, but we made a poll on Slack and the majority decided to wipe the `advancedSearch` method out.

![selection_144](https://cloud.githubusercontent.com/assets/5023426/21221839/45ab770a-c2bf-11e6-92ec-c097655c97d1.png)